dev/tools/frontools/config/themes.json

```
@@ -1,10 +1,35 @@
 {
   "sd": {
-    "src"             : "vendor/somethingdigital/magento2-theme-bryantpark",
-    "dest"            : "pub/static/frontend/SomethingDigital/bryantpark",
-    "locale"          : ["en_US"],
-    "lang"            : "scss",
-    "postcss"         : ["plugins.autoprefixer()", "plugins.postcssInlineSvg()", "plugins.postcssObjectFitImages()"],
-    "ignore"          : ["node_modules/**"]
+    "src": "vendor/somethingdigital/magento2-theme-bryantpark",
+    "dest": "pub/static/frontend/SomethingDigital/bryantpark",
+    "locale": [
+      "en_US"
+    ],
+    "lang": "scss",
+    "postcss": [
+      "plugins.autoprefixer()",
+      "plugins.postcssInlineSvg()",
+      "plugins.postcssObjectFitImages()"
+    ],
+    "ignore": [
+      "node_modules/**"
+    ]
+  },
```

Pro tip - this will let us skip building this theme on `gulp styles`

```suggestion
    ],
    "skipBuild": true
  },
```